CASMTRIAGE-6686: Document hms-discovery timeout workaround when many switches are missing (#4966)
Cray-HPE/docs-csm: troubleshooting/known_issues/hms_discovery_timeout_due_to_missing_switches.md

# hms-discovery Timeout Due to Missing Switches

## Overview

On systems with a large number of non-existent network switches, it has been observed that the hms-discovery job may fail to finish due to spending too much time trying to communicate with the non-existent switches.
This may happen when incrementally building up a new system and running CSM prior to completion of the full hardware installation.

## Symptoms and Diagnosis

> Should hms-discovery be found to not complete in such an environment, this specific problem can be diagnosed as follows:

Reviewer: ```suggestion
If hms-discovery does not complete in such an environment, this specific problem can be diagnosed as follows:
```
